### PR TITLE
FX-1738: only exclude collection hubs from the iOS app

### PR DESCRIPTION
**Pull request by xtina-starr**
Addresses: [FX-1738](https://artsyproduct.atlassian.net/browse/FX-1738) and [FX-1739](https://artsyproduct.atlassian.net/browse/FX-1739)

This remove the rule to exclude all collections pages from being shown in the iOS app ~and scopes it down to just exclude department (hub) collections~. I  addresses some merge conflicts in this PR so I'd appreciate it the reviewers took extra care to look at that as well.

Once this is merged we will need to update the dependencies that use this library:
- [ ] force
- [ ] artsy-wwwify
- [ ] sailthru *

*Updates to Sailthru requires you to notify Sailthru support and provide them with the updated `apple-app-site-association.json` file.

### CODE REVIEW

File: apple-app-site-association.json [hunk 1]
```diff
@@ -25,21 +27,22 @@
...
           "NOT /click/*/aHR0cHM6Ly9hcnRzeS50eXBlZm9ybS5j*",
           "NOT /click/*/aHR0cHM6Ly9hcHAuYWRqdXN0LmNv*",
           "NOT /users/auth/facebook/callback*",
+          "NOT **",
           "NOT *L2FydGljbGUv*",
+          "NOT **",
           "NOT *L3ZpZGVv*",
           "NOT *L25l*",
           "NOT *L25ld3Mv*",
+          "NOT **",
```
Comment (ashfurrow) [inline apple-app-site-association.json]: I don't know why these keep showing up in the file, but we should remove `NOT **` entries.

**Review by ashfurrow — COMMENTED** [review]
Looks good 👍 Just one comment.

Note that for sequencing, we will want to do the following:

1. Release Collections for iOS
1. Wait a bit for users to upgrade (about a week, I'd say?)
1. Merge this PR and deploy

File: apple-app-site-association.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "activitycontinuation": {
+  "activity ntinuation": {
```
Comment (pepopowitz) [inline apple-app-site-association.json]: This seems like an accidental deletion of some characters.

Comment (xtina-starr) [reply apple-app-site-association.json]: Aww good catch!

File: apple-app-site-association.json [hunk 3]
```diff
@@ -25,8 +25,6 @@
           "NOT /2016-year-in-art",
           "NOT /venice-biennale*",
           "NOT /gender-equality*",
-          "NOT /collections",
```
Comment (sweir27) [inline apple-app-site-association.json:28]: Did we build this index view in the app? I _think_ we'll still want users to see the web version of this (but could totally be missing something).

**Review by sweir27 — COMMENTED** [review]
Just one quick comment, other than that looks good!

File: apple-app-site-association.json (hunk 3 above)
Comment (ashfurrow) [reply apple-app-site-association.json:28]: +1 I think we still want to keep this line for now.

Comment (xtina-starr) [reply apple-app-site-association.json:28]: Good catch. I just corrected it.

File: apple-app-site-association.json (hunk 1 above)
Comment (dleve123) [reply apple-app-site-association.json]: This is a bug in the `yarn build` script for this repo. Just got these extraneous `NOT **` after running it.

Comment (ashfurrow) [reply apple-app-site-association.json]: Yeah, I need to dig into that 😅

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.1.0`](https://github.com/artsy/artsy-eigen-web-association/releases/tag/v1.1.0) :rocket:

**Pull request merged**